modules/imaging_uploader/js/imaging_uploader.js

```diff
@@ -155,3 +155,302 @@ $(function () {
+    var uploadId = uploadProgress.getUploadId();
+    
+    $.post(
+        "/AjaxHelper.php?Module=imaging_uploader&script=getUploadSummary.php",
+        {uploadId: uploadId, summary: summary},
+        function (data) {
+            uploadProgress.setProgressFromServer(data);
+            // If the pipeline is still running, start polling
+            // If the pipeline is not running, end the polling (if any was started)
+            setServerPolling(uploadProgress.isInserting());
+            $('textarea[name=UploadLogs]').val(uploadProgress.getProgressText());
+        }    
+    );  // post call
+}
+
+/**
+ * Starts/stops polling on the server.
+ */
+function setServerPolling(poll) {
+    // If polling should be stopped
+    if(!poll) {
+        // Stop issuing POST requests (if any polling was taking place)
+        if(setServerPolling.getSummaryInterval) {
+            clearInterval(setServerPolling.getSummaryInterval);
+            setServerPolling.getSummaryInterval = null;
+        }
+        // Stop updating the series of dots string (if such an update was
+        // going on) 
+        if(setServerPolling.dotUpdateInterval) {
+            clearInterval(setServerPolling.dotUpdateInterval);
+            setServerPolling.dotUpdateInterval= null;
+        }
+        // Stop updating the animated char (if such an update was
+        // going on) 
+        if(setServerPolling.animatedCharInterval) {
+            clearInterval(setServerPolling.animatedCharInterval);
+            setServerPolling.animatedCharInterval = null;
+        }  
+    // If polling should begin  
+    } else {
+        // If there were no POST requests being issued, start issuing some.
+        if(!setServerPolling.getSummaryInterval) {
+            setServerPolling.getSummaryInterval = setInterval(
+                monitorProgress,
+                5000
+            )
+        }
+
+        // If there were no updates to the string of dots, start updating
+        if(!setServerPolling.dotUpdateInterval) {
+            setServerPolling.dotUpdateInterval = setInterval(
+                function() {
+                    uploadProgress.updateDots(); 
+                    $('textarea[name=UploadLogs]').val(uploadProgress.getProgressText());
+                },
+                3000
+            )
+        }
+        
+        // If there were no updates to the animated chars, start updating
+        if(!setServerPolling.animatedCharInterval) {
+            setServerPolling.animatedCharInterval = setInterval(
+                function() {
+                    uploadProgress.updateAnimatedCharIndex();
+                    $('textarea[name=UploadLogs]').val(uploadProgress.getProgressText());
+                },
+                250
+            )
+        }
+    }
+}
+
+/**
+ * Invokes wen the DOM is ready. Will repond to clicks in the MRI upload table by
+ * updating the upload logs for that MRI scan in the logs table.
+ */
```

Comment:
It does work and it is great. But somehow clicking once works (it gives some messages), clicking another time (and any number of times afterwards) goes back to the main message of <select a row in the table below to view the upload logs>. I have to either refresh the page or switch from Detailed to Summary views to get the messages back.